### ISSUE TITLE
NS_ERROR_FILE_TOO_BIG from getTempDir.js when trying to install new script

### ISSUE THREAD

**Issue by reibuehl** [greasemonkey/greasemonkey#2069]
Hi,

I have the following problem when trying to install new scripts: I get the install script bar, but when I click the Install button, nothing happens. I just get the following error message in the Browser Console:

NS_ERROR_FILE_TOO_BIG: Component returned failure code: 0x8052000f (NS_ERROR_FILE_TOO_BIG) [nsIFile.createUnique]         getTempDir.js:13

Greasemonkey is enabled as a plugin and also internally. If I try to create a new script manually, I see the same behaviour when I click the OK button that should open the editor after entering the script metadata.

I use FireFox 34 with Greasemonkey 2.3 on Windows 7 x64.

**Comment by noahm** [comment]
I just ran into this issue with GM3.2, so whatever was supposed to kick off the cleanup isn't working as expected.

**Comment by arantius** [comment]
It would be useful if you could report a new issue, and be sure to describe it exactly in as much detail as possible.  What were you doing, what did you click exactly?  Copy/paste the exact/entire error line from the console.  Thanks.

**Comment by noahm** [comment]
I can copy/paste the OP into a new issue if you really want me to, but it seems like a waste.

I found I was unable to install (from a gist) or create new userscripts due to some sort of silent failure. When opening the browser console I see the same NS_ERROR_FILE_TOO_BIG error. I checked my %TEMP% folder and found 10k gm-temp folders. Manually deleting those folders resolved the issue.

I have GM 3.2 and Firefox 38.0.5
